public/toolkit/styles/utilities/_positioning.scss

```diff
@@ -102,7 +102,7 @@ $position-default: null !default;
   display: inline-block !important;
 }
 
-.a-center {
+@mixin a-center {
```

Comment:
we weren't using this anywhere else. Wondering if I should pull the `text-align:center` into this too. display + margin work for buttons, and text-align works for links

Reply:
I'm fine with bringing text-align:center into this mixin